frontend/src/text_resources.js

```diff
@@ -128,63 +128,54 @@ let LOCALIZE = new LocalizedStrings({
-          bullet3:
-            "Use only the information provided in the emails and the background information. Do not make any inferences based on the culture of your own organization. Avoid making assumptions that are not reasonably supported by either the background information or the emails.",
-          bullet4: "You can answer the emails in any order you want.",
-          bullet5: "You are responsible for managing your own time."
-        },
         testInstructions: {
           title: "Test instructions",
           para1:
-            "First, read the background information which describes your position and the fictitious organization in which you work. Then proceed to the inbox where you can read the emails you received and take actions to respond to them as though you were a manager within the fictitious organization.",
+            "When you start the test, first, read the background information which describes your position and the fictitious organization in which you work. Then proceed to the inbox where you can read the emails you received and take actions to respond to them as though you were a manager within the fictitious organization.",
           step1Section: {
             title: "Step 1 - Responding to emails",
             description:
               "You can respond to the emails you receive in two ways: by sending an email or by adding tasks to your task list. A description of both methods of responding is presented below, followed by examples.",
             part1: {
-              title: "Sending an email",
+              title: "Example of an email you have received:",
+              para1:
+                "Two options are provided below to demonstrate different methods of responding to the email. As previously mentioned, you can choose one of the two options presented or a combination of the two. Note that the responses provided were not evaluated for how well they demonstrate the competency targeted in the email. They are presented here only to illustrate how to use each of the two methods of responding."
+            },
+            part2: {
+              title: "1. Responding with an email response",
               para1:
                 "You can send an email in response to one you received in your inbox. The written responses should reflect how you would respond as a manager.",
               para2:
                 "You can send an email using the following options: reply, reply all, forward, or write a new email. If you choose to write a new email, you will have access to a directory with all of your contacts. You can write as many emails as you like in response to an email you received."
             },
-            part2: {
-              title: "Adding a task to the task list",
+            part3: {
+              title: "2. Adding a task to the task list",
```

Comment:
Only one small comment. The numbers on the title here and above were removed on purpose for accessibility reasons. Can we leave them as is?

`Responding with an email response`
and
`Adding a task to the task list`

Reply:
Changed back in 65f6112